server/services/curiousService.ts

```diff
@@ -18,11 +18,11 @@ export default class CuriousService {
   ) {}
 
   async getPrisonerSupportNeeds(prisonNumber: string, username: string): Promise<PrisonerSupportNeeds> {
-    const systemToken = await this.hmppsAuthClient.getSystemClientToken(username)
     const prisonNamesById = await this.prisonService.getAllPrisonNamesById(username)
+    const curiousClientToken = await this.hmppsAuthClient.getCuriousClientToken()
```

Comment:
This is the key change in respect of the Curious Service class. Instead of getting the UI's system token via `hmppsAuthClient.getSystemClientToken`, we get the Curious token via the new method `this.hmppsAuthClient.getCuriousClientToken`